src/Rules/Compile.hs

```diff
@@ -39,3 +47,35 @@ needCompileDependencies :: Context -> Action ()
 needCompileDependencies context@Context {..} = do
     when (isLibrary package) $ need =<< return <$> pkgConfFile context
     needContext =<< contextDependencies context
+
+needGenerated :: Context -> FilePath -> Action ()
+needGenerated context origFile = go Set.empty
+  where
+    go :: Set.Set String -> Action ()
+    go done = withTempFile $ \outFile -> do
+        let mode = Cc FindMissingInclude $ stage context
```

Comment:
Yes, thanks for noticing! (I refactored the code a bit but forgot to change the name)